docs/05.PORT-API.md

```diff
@@ -0,0 +1,210 @@
+# Reference
+
+## Termination
+
+It is questionable whether a library should be able to terminate an application. Some of the API functions may signal an error (ex.: cannot allocate memory). So, we keep the termination approach with this port function.
```

Comment:
Only some of the API functions can signal an error ... for the others, we keep ...